pymc3/backends/base.py

```diff
@@ -92,7 +93,7 @@ def _set_sampler_vars(self, sampler_vars):
 
         self.sampler_vars = sampler_vars
 
-    def setup(self, draws, chain, sampler_vars=None):
+    def setup(self, _draws, _chain, sampler_vars=None):
```

Comment:
Is this just to silence the linter? If so, a comment like ` # pylint: disable=unused-argument` should work as well, and I think that would be preferable.

Reply:
Good point. I will fix that.